themes/bootstrap3/templates/ajax/itemLocationList.phtml

```diff
@@ -0,0 +1,12 @@
+<?php $availabilityStatus = $this->availabilityStatus(); ?>
+<?php foreach ($this->locationList as $location): ?>
+  <div class="groupLocation">
+    <span class="<?= $availabilityStatus->getClass($location['availability'])?>">
+      <?=$this->icon($availabilityStatus->getIcon($location['availability']))?>
+      <?=$this->escapeHtml($location['location'])?>
+    </span>
+  </div>
+  <div class="groupCallnumber">
+    <?=$this->escapeHtml($location['callnumber'])?>
```

Comment:
`$location['callnumber']` can be HTML if a callnumberHandler is used. So this should not be escaped here.
(This is causing the failure of the CallnumberBrowseTest)

Reply:
I fixed this.